app/_locales/en/messages.json

```diff
@@ -6231,7 +6231,7 @@
     "message": "To use wallet notifications, we use a profile to sync some settings across your devices. $1"
   },
   "turnOnMetamaskNotificationsMessageThird": {
-    "message": "You can turn off notifications at any time in $1"
+    "message": "You can turn off notifications at any time in the $1"
```

Comment:
(nit - you can ignore), maybe a description might be handy to inform what the `$1` is used for. Realistically we will be re-working this modal in the very-near future to add the notification settings page.